app/(main)/layout.tsx

```diff
@@ -75,7 +75,7 @@ export default function RootLayout({
         <GoogleAnalytics ga_id={process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS} />
       ) : null}
       <Providers>
-        <body>
+        <body className="m-0">
           <div className="print:hidden">
```

Comment:
remove this extra div and apply the CSS rule directly inside of `<Layout />`. You only use extra divs for structure and layout purposes.